fedimint-core/src/core/server.rs

```diff
@@ -204,14 +211,17 @@ where
     }
 
     /// Blocks until a new `consensus_proposal` is available.
-    async fn await_consensus_proposal(&self, dbtx: &mut DatabaseTransaction<'_>) {
```

Comment:
Do we want IsolatedDatabaseTransaction here? It is exposing outside the implementation detail of modules getting isolated. Is it an attempt to make sure we never forget about isolation on the caller side?

Reply:
Oh. I thought I'm looking at a different PR of yours ...